database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -240,18 +248,19 @@ private void setResultSetMetadata() throws Exception {
     connectionProperties.putAll(dbSinkConfig.getConnectionArguments());
     try (Connection connection = DriverManager.getConnection(connectionString, connectionProperties)) {
       executeInitQueries(connection, dbSinkConfig.getInitQueries());
-      try (Statement statement = connection.createStatement();
-           // Run a query against the DB table that returns 0 records, but returns valid ResultSetMetadata
-           // that can be used to construct DBRecord objects to sink to the database table.
-           ResultSet rs = statement.executeQuery(String.format("SELECT %s FROM %s WHERE 1 = 0",
-                                                               dbColumns,
-                                                               dbSinkConfig.getEscapedTableName()))
-      ) {
-        ResultSetMetaData resultSetMetadata = rs.getMetaData();
-        columnTypes.addAll(getMatchedColumnTypeList(resultSetMetadata, columns));
-      }
-    }
 
+
+        try (Statement statement = connection.createStatement();
```

Comment:
Same here.. keep indentation as it was before everywhere in the PR. So only change would be of tablename.

Reply:
got it. Thanks. I know know how your doing your comparisons and what to look for specifically.